apps/spruce/src/pages/version/TestAnalysis/index.tsx

```diff
@@ -129,24 +131,38 @@ const TestAnalysis: React.FC<TestAnalysisProps> = ({ versionId }) => {
           <FilterContainer>
             <TextInputWithValidation
               aria-labelledby="test-failure-search-label"
+              defaultValue={testName}
               disabled={!hasResults}
               id="test-failure-search-input"
               label="Search Test Failures"
-              onSubmit={setTestName}
+              onSubmit={(value) => {
```

Comment:
Would it be possible to make it more clear what search is applied? Something like this could work
<img width="356" alt="image" src="https://github.com/user-attachments/assets/36b86322-04f0-4c48-b917-3e2d94db8fbf">

Reply:
It should be clear from the filter inputs which is what I tried to do with the defaultValue from your below comment.
<img width="977" alt="image" src="https://github.com/user-attachments/assets/1d6add83-93a2-4270-a771-a2d085e42da5">

I think repeating it twice might be too much. Do you think its necessary?

Reply:
I kind of agree, it's hard for me to tell when filters are applied. I feel like there should be filter badges here

It also seems like Clear Filters doesn't actually clear the inputs which makes it look like filters are still applied

Reply:
I don't think filter badges are necessarily the best choice here because we already use the badges in the comboboxes so having to manage badges in 2 places is confusing. And the test filter input only allows one input while the badge pattern is more commonly used when we allow users to apply multiple filters like we do on the project health page or task history pages.